doc/Procedures.md

```diff
@@ -12,3 +12,30 @@ git tag $version
 git push origin $version
 ```
 - Update [releases](https://github.com/dnanexus/dxfuse/releases) github page, use the `Draft a new release` button, and upload executables.
+
+## Track usage with Splunk logs
+
+Launch information is automatically recorded in the `user-agent` field of `/system/greet` API call when creating a new dxFUSE session, with the following fields included:
```

Comment:
may be give an example of the user-agent string (with fake user ids) and document Mode variables.

Reply:
https://github.com/dnanexus/dxfuse/pull/108/commits/f7da12654c9506f8c65c62f48961840604a79dcd
also removed workspace